src/components/TextField/TextField.tsx

```diff
@@ -432,6 +432,7 @@ export function TextField({
     pattern,
     inputMode,
     type: inputType,
+    rows: multiline ? multiline : undefined,
```

Comment:
multiline can be a boolean according to the props. Is passing a boolean to a textarea's rows attribute valid?

Reply:
I'm guessing we can probably make this 1 as a default when multiline as this is what it renders as when true